### ISSUE TITLE
"Cannot open file to write"

### ISSUE THREAD

**Issue by mustafasyed123** [evancohen/smart-mirror#725]
I'll be frank guys, I have no idea what I'm doing. I'm trying to customize my smart-mirror with modules and every time I try to paste in the code for a module into config.json and save it, I get an error saying "can't open file to write". Also stupid question: are config.js and config.json the same thing? I really hope so.

**Comment by evancohen** [comment]
No, these files are not the same. `config.js` is for reading the `config.json` file in a reusable way. You shouldn't be manually editing the `config.json` file - it's auto-generated by the remote config tool.

Is it possible that you used `sudo` when you should not have and locked the permissions of the file? If so you can 
```
sudo chmod 777 config.json
```
